File: 1730-special-array-with-x-elements-greater-than-or-equal-x/special-array-with-x-elements-greater-than-or-equal-x.py
class Solution:
    def specialArray(self, nums: List[int]) -> int:
        maxi = max(nums)
        for i in range(maxi+1):
            temp = 0
            for j in nums:
                if(j>=i):
                    temp+=1
            if(temp==i): 
                return i
        return -1

File: 1730-special-array-with-x-elements-greater-than-or-equal-x/special-array-with-x-elements-greater-than-or-equal-x.cpp
class Solution {
public:
    int specialArray(vector<int>& nums)
    {
        int maxi = *max_element(nums.begin(),nums.end());
        for(int i=0;i<=maxi;i++)
        {
            int temp=0;
            for(int j=0;j<nums.size();j++)
            {
                if(nums[j] >= i) temp++;
            }
            if(temp==i) return i;
        }
        return -1;
    }
};